webapp/package.json

```diff
@@ -23,7 +23,8 @@
     "@uppy/webcam": "^1.8.4",
     "@uppy/xhr-upload": "^1.6.10",
     "@vueuse/components": "^10.7.2",
-    "bootstrap": "^4.5.3",
+    "boostrap": "^2.0.0",
```

Comment:
:eye: think this one needs to be removed! Thankfully this is one where npm have sat on the name to prevent malicious use